components/action/GeometricBackgroundSmall.tsx

```diff
@@ -0,0 +1,81 @@
+import Image, { StaticImageData } from 'next/image';
+import React from 'react';
+import {
+  elipse118,
+  elipse117,
+  rectangle4439,
+  elipse119,
+  Rectangle4401,
+} from '@/public/assets/index';
+
+type GeometricItem = {
+  src: StaticImageData;
+  alt: string;
+  width: number;
+  height: number;
+  position: string;
+};
+
+const items: GeometricItem[] = [
```

Comment:
Move this into a separate component so it's reusable in multiple places.

Reply:
resolved